packages/postgraphile-core/__tests__/integration/__snapshots__/mutations.test.js.snap

```diff
@@ -203,7 +203,7 @@ Object {
         },
       },
       "e": Object {
-        "cursor": "WyJlbWFpbF9hc2MiLFsiam9obm55LmJveS5zbWl0aEBlbWFpbC5jb20iLDkwMDBdXQ==",
+        "cursor": "WyJlbWFpbF9hc2MiLFsiam9obm55LmJveS5zbWl0aEBlbWFpbC5jb20iXV0=",
```

Comment:
`["email_asc",["johnny.boy.smith@email.com",9000]]` becomes `["email_asc",["johnny.boy.smith@email.com"]]` because email is unique